src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: $star-size;
+    width: $star-size;
+    background-image: url($star-default);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:not(:last-of-type) {
+    margin-right: $star-spacing;
```

Comment:
The variable `$star-spacing` is used here but not defined. Please ensure that this variable is defined in a `vars.scss` file or imported correctly to avoid compilation errors.